FirstProject/assets/evaluation.less

```diff
@@ -0,0 +1,54 @@
+//Evaluation less file
+
+.row {
+
+  .navigation_bar {
+    border-bottom: 2px solid darkgrey;
+    padding-top: 12px;
+    padding-bottom: 14px;
+  }
+
+  .logo_image {
+    width: 40%;
+  }
+}
+
+.col-md-4 {
+
+  .footer_description {
+    padding: 25px 0 12px 0;
```

Comment:
Are you referring to an other way to write it?

An example.  padding: 10px 12px; (top and bottom 10px, right and left 12px)